### PR TITLE
Fix discourse category creation

### PR DESCRIPTION
**Pull request by mario**
This is a simple copy&paste job for now to fix the fact that discourse category is not created upon integration approval.

Also, since I can't test this - please do it. Thanks!

### CODE REVIEW

**Comment by mario** [comment]
Rebased.

File: nextcloudappstore/scaffolding/forms.py [hunk 1]
```diff
@@ -86,6 +87,28 @@ class IntegrationScaffoldingForm(Form):
                                         ' integration '
                                         'does. Can contain Markdown.'))
 
+    def _create_discourse_category(self, app_id: str) -> None:
+        url = '%s/categories?api_key=%s&api_username=%s' % (
+            settings.DISCOURSE_URL.rstrip('/'),
+            settings.DISCOURSE_TOKEN,
```
Comment (MorrisJobke) [inline nextcloudappstore/scaffolding/forms.py:93]: As of now token and user are `None` ... let me set some up there first :)

Comment (mario) [reply nextcloudappstore/scaffolding/forms.py:93]: Ok. This was used in another place as well...

Comment (MorrisJobke) [reply nextcloudappstore/scaffolding/forms.py:93]: Most likely on an non-authenticated API I think. Do you have a link?

Comment (MorrisJobke) [reply nextcloudappstore/scaffolding/forms.py:93]: Found it ... Anyways - I created one.

Comment (mario) [reply nextcloudappstore/scaffolding/forms.py:93]: :)

**Pull request merged**